bcda/auth/plugin/alpha.go

```diff
@@ -188,8 +189,31 @@ func (p *AlphaAuthPlugin) RequestAccessToken(params []byte) (jwt.Token, error) {
 }
 
 // lookup token and set active flag to false
-func (p *AlphaAuthPlugin) RevokeAccessToken(token string) error {
-	return errors.New("not yet implemented")
+func (p *AlphaAuthPlugin) RevokeAccessToken(tokenString string) error {
+
+	backend := auth.InitAuthBackend()
+	claims := backend.GetJWTClaims(tokenString)
```

Comment:
would we prefer to use `DecodeAccessToken()` and the `CustomClaims` struct in this plugin now?